pcsx2/gui/RecentIsoList.cpp

```diff
@@ -178,6 +178,17 @@ void RecentIsoManager::InsertIntoMenu( int id )
 
 	if ( !exists )
 		curitem.ItemPtr->Enable( false );
+	else
+		curitem.ItemPtr->Enable( !g_Conf->AskOnBoot );
```

Comment:
I think the local variable ``exists`` is also pointless. The function's name is descriptive on it's own.